app/src/main/res/values/strings.xml

```diff
@@ -596,4 +596,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="wallpaper_set_unsuccessfully">Something went wrong. Could not set the wallpaper</string>
   <string name="setting_wallpaper_dialog_title">Set as Wallpaper</string>
   <string name="setting_wallpaper_dialog_message">Setting Wallpaper. Please wait…</string>
+
+  <string name="cannot_open_location_settings">Could not open location settings. Please turn location on manually</string>
```

Comment:
Should we be saying GPS instead of location as that's what we're looking for? 🤔

Reply:
Yeah but the settings page that will open will be of `Location`. We could show this instead:`Failed to open location settings. Please turn on location and set it to High Accuracy`.